apps/app/components/Preview/index.tsx

```diff
@@ -263,6 +263,62 @@ export const PreviewComponent: React.FC = () => {
     )
   }, [userProps, detail])
 
+  const mailAddress: string = useMemo(
+    () => userProps?.defaultAddress ?? 'unknown',
+    [userProps]
+  )
+
+  const toMessage = useMemo(() => {
+    const isOfficeMail = OFFICE_ADDRESS_LIST.some(
+      (address) => detail?.from.address === address
+    )
+
+    if (isOfficeMail && detail && detail.to === null) {
+      return [
+        {
+          address: mailAddress,
+        },
+      ]
+    }
+
+    return detail?.to || []
+  }, [detail])
+
+  const avatarList = useMemo(() => {
+    if (!detail) return []
+    const exists: Array<string> = []
```

Comment:
If you need to remove weight, you can use [Set](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set).

Like: 
```tsx
return Array.from(
  new Set(
    [detail.from, ...toMessage, ...(detail.cc || []), ...(detail.bcc || [])]
  )
)
```